charts/ocis/docs/values.adoc.yaml

```diff
@@ -812,15 +848,19 @@ extraResources: []
 
 # per-service configuration.
 services:
-  # -- APP PROVIDER service. Not used if `features.appsIntegration.enabled` equals `false`.
+  # -- COLLABORATION service. Not used if `features.appsIntegration.enabled` equals `false`.
   # @default -- see detailed service configuration options below
-  appprovider:
+  collaboration:
```

Comment:
We should somehow highlight that this is a breaking change

Reply:
if this chart would be released, it would be highlighted in the changelog. Please note that this chart was not released since September 2023 (https://github.com/owncloud/ocis-charts/releases/tag/v0.5.0).

See also #78 and #325